plugin-hrm-form/src/components/resources/resourceview/ViewResource.tsx

```diff
@@ -0,0 +1,283 @@
+    const eligibilityMaxAge = getSingleStringVal(attributes, 'eligibilityMaxAge');
+    if (eligibilityMinAge && eligibilityMaxAge) {
+      return `${eligibilityMinAge} - ${eligibilityMaxAge} years`;
+    }
+    return 'N/A';
+  };
+
+  const handlePrimaryLocation = (attributes: Object) => {
+    const county = getSingleStringVal(attributes, 'primaryLocationCounty');
+    const city = getSingleStringVal(attributes, 'primaryLocationCity');
+    const province = getSingleStringVal(attributes, 'primaryLocationProvince');
+    const postalCode = getSingleStringVal(attributes, 'primaryLocationPostalCode');
+    const phone = getSingleStringVal(attributes, 'primaryLocationPhone');
+    // eslint-disable-next-line prefer-named-capture-group
+    const formattedPhone = phone.replace(/(\d{3})(\d{3})(\d{4})/, '$1-$2-$3');
+
+    return `${county}, ${city}\r\n${province}, ${postalCode}\r\n${formattedPhone}`;
+  };
+
+  const handleDescriptionInfo = descriptionObj => {
+    return descriptionObj[0].info.text;
+  };
+  return (
+    <ResourceViewContainer>
+      <Column>
+        <Box marginTop="10px" marginBottom="10px">
+          <SearchResultsBackButton
+            text={<Template code="SearchResultsIndex-BackToResults" />}
+            // eslint-disable-next-line no-empty-function
+            handleBack={() => navigateToSearch()}
+          />
+        </Box>
+        <ViewResourceArea>
+          {error && ( // TODO: translation / friendlyisation layer
+            <>
+              <ResourceTitle>
+                <Template code="Resources-LoadResourceError" />
+              </ResourceTitle>
+              <p>{error.message}</p>
+            </>
+          )}
+          {resource && (
+            <>
+              <ResourceTitle>{resource.name}</ResourceTitle>
+              <ResourceIdCopyButton resourceId={resource.id} />
+              {resource.attributes && (
+                <ResourceAttributesContainer>
+                  {/* FIRST COLUMN */}
+                  <ResourceAttributesColumn>
+                    <ResourceAttribute
+                      description="Status"
+                      content={getSingleStringVal(resource.attributes, 'status')}
+                    />
+                    <ResourceAttribute
+                      description="Taxonomy Code"
+                      content={getSingleStringVal(resource.attributes, 'taxonomyCode')}
+                    />
+                    <ResourceAttribute
+                      description="Details"
+                      content={handleDescriptionInfo(resource.attributes.description)}
+                    />
+                  </ResourceAttributesColumn>
+
+                  {/* SECOND COLUMN */}
+                  <ResourceAttributesColumn>
+                    <ResourceAttributeWithPrivacy
+                      isPrivate={getSingleStringVal(resource.attributes.mainContact, 'isPrivate') === 'false'}
```

Comment:
```suggestion
                      isPrivate={getSingleStringVal(resource.attributes.mainContact, 'isPrivate') === 'true'}
```

Did you flip it for testing? everything is showing as private now and I don't think any of the test data is flagged private